tdp/cli/commands/init.py

```diff
@@ -31,19 +36,49 @@
     multiple=True,
     help="Path to TDP variables overrides. Can be used multiple times. Last one takes precedence.",
 )
+@click.argument(
+    "alembic_ini_path",
+    envvar="ALEMBIC_CONFIG",
+    required=True,
+    type=click.Path(exists=True, resolve_path=True, path_type=Path),
+)
 @collections_option
-@database_dsn_option
+@database_dsn_option(create_engine=False)
 @validate_option
 @vars_option(exists=False)
 def init(
     overrides: tuple[Path],
     collections: Collections,
-    db_engine: Engine,
+    alembic_ini_path: Path,
+    db_engine: str,
     validate: bool,
     vars: Path,
 ):
     """Initialize the database and the TDP variables."""
-    init_database(db_engine)
+    engine = create_engine(db_engine)
+    with engine.connect() as connection:
+        alembic_config = Config(
+            file_=str(alembic_ini_path),
```

Comment:
Why setting `path_type=Path` for the option if the goal is to cast it as a string right after?